examples/trait-incrementer/traits/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "increment-traits"
```

Comment:
Small thing here, but can you make sure that the name of the package and folder are the same? It makes it more intuitive when building contracts with `--manifest-path`.